src/components/core/components/CardKeyValue/CardKeyValue.tsx

```diff
@@ -39,14 +39,14 @@ export default function CardKeyValue(props: Props) {
       <TableBody>
         {rows.map((row) => (
           <TableRow key={row.key}>
-            <StyledTableCell hideDivider={hideDivider}>
-              <Typography variant="body1" color="textSecondary" noWrap>
+            <StyledTableCell $hideDivider={hideDivider}>
```

Comment:
$hideDivider property does not exists

Reply:
the $ makes it so that the react dom does not yell at us about it

Reply:
react dom as i said before

Reply:
we can fix it directly in the styled component

Reply:
> we can fix it directly in the styled component

Thank you i fixed it